src/main/app/src/pages/koulutus/KoulutusForm/KoulutuksenEPerusteTiedot/ValitseOsaamisalaBox.tsx

```diff
@@ -156,6 +185,11 @@ export const ValitseOsaamisalaBox = ({
                 </Anchor>
               ),
             },
+            {
+              title: t('koulutuslomake.osaamisalanLaajuus'),
+              description: osaamisalaLaajuus,
+              suffix: t('yleiset.osaamispistetta'),
+            },
```

Comment:
Tätä ei varmaan haluta rendata jos osaamisalaLaajuutta ei ole

Reply:
Jäin vielä miettimään tätä. Nythän se näkyy siis tälleen: 
![image](https://user-images.githubusercontent.com/63100253/121297728-71693980-c8fb-11eb-957a-0798ab893acd.png)
Eli jos laajuutta ei ole, niin sen kohdalla on viiva samalla tavalla kuin Voimaantulon kohdalla. Mut voin myös poistaa ton laajuuden, jos on silleen parempi.

Reply:
Kyl tääkin on hyvä :+1:

Jäin nyt miettimään että näkyyhän tässä tilanteessa sama numero / teksti kuin oppijalla? Eli varmistaisin että jos tuossa ei pystytä päättelemään laajuutta niin mitä indeksoija siihen tallentaa, vai tallentaako mitään jolloin oppijallakaan ei kerrota mikä on ko. osaamisalan laajuus?

Reply:
Nuo kannattaa varmasti tehdä samalla tavalla. En pidä tätä kauhean olennaisena kysymyksenä, mutta mun mielestä on myös ihan hyvä, että esitetään eksplisiittisesti, kun jotain arvoa ei löydy. Ei varmaan kannata lähteä tätä muuttamaan, kun tuo on ollut noin muissa kohdissa jo yli vuoden eikä kukaan ole valittanut mitään :smile:

Reply:
> Jäin nyt miettimään että näkyyhän tässä tilanteessa sama numero / teksti kuin oppijalla? Eli varmistaisin että jos tuossa ei pystytä päättelemään laajuutta niin mitä indeksoija siihen tallentaa, vai tallentaako mitään jolloin oppijallakaan ei kerrota mikä on ko. osaamisalan laajuus?

Indeksoijaan on toteutettu sama oletuslaajuusarvon asettaminen kuin täällä frontin puolella, koska se haetaan molemmissa paikoissa eperusteista. Eli periaatteessa mekanismin pitäisi olla sama. :) Jos sitä laajuusarvoa ei jostain syystä löydy, niin nythän oppijan puolella tulee teksti, että "Ei opintojenlaajuutta", mut mitä nyt testailin untuvalla, niin siellä näyttäisi olevan virkailijan ja oppijan puoli synkassa eli samat arvot näkyy molemmissa.